pkg/output/stdout.go

```diff
@@ -146,7 +147,11 @@ func (s *Stdoutput) printResult(resp ffuf.Response) {
 	if s.config.Quiet {
 		s.resultQuiet(resp)
 	} else {
-		s.resultNormal(resp)
+		if s.config.AltOutput {
```

Comment:
AltOutput doesn't exist in the config struct anymore, so this fails to compile.
```suggestion
		if s.config.UrlOutput {
```